change-author-email-of-commits.md

```diff
@@ -0,0 +1,68 @@
+# Changing author email of commits in bulk
+
+Sometimes, you might need to update the author or committer information in Git commit history. This could be necessary if, for example, you've been using incorrect or outdated information.
+
+Here's a step-by-step guide on how to rewrite commit history in Git to update author and committer information:
+
+## Prerequisites
+
+Before proceeding, make sure you have the following:
+
+- Git installed on your system.
+- Basic understanding of Git commands.
+
+## Steps
+
+1. **Identify the Incorrect Information:**
+
+
+   Determine the incorrect author or committer information that needs to be updated. This typically includes the email address associated with the commits.
+
+
+2. **Set up Variables:**
+
+Replace "incorrect_email@example.com" with the incorrect email address, "new_author_name" with the desired new author name, and "new_email@example.com" with the desired new email address.
+
+   ```bash
+   WRONG_EMAIL="incorrect_email@example.com"
+   NEW_NAME="new_author_name"
+   NEW_EMAIL="new_email@example.com"
+   ```
+3. **Run the Command:**
+
+Execute the following command in your Git repository:
+
+   ```bash
+WRONG_EMAIL=${1:-"pasindu.dinal@qlub.io"}
```

Comment:
I think it'll be better to mention what 1:- here is doing, how the script can be invoked if we're passing arguments from outside